docs/guides/create_update_solution.md

```diff
@@ -0,0 +1,339 @@
+---
+ hide:
+  - toc
+---
+!!! abstract "Remember"
+
+    You have to choose the platform and project you want to work. Contact your babylon admin to know more about your options.
+
+    In this example, we will use
+
+    * context_id: `test`
+    * platform_id: `dev`
+
+To deploy a complete solution, you can declare its configuration in yaml files corresponding
+to specific deployment type. Each file contains general information about the deployment:
+
+```yaml
+kind: Organization
+namespace:
+  state_id: "{{state_id}}"
+  context: demo
+  platform:
+    id: dev
+    url: https://dev.api.cosmotech.com/phoenix/v3-0 
+```
+The `kind` key must be one of these: Organization, Solution, Workspace, WebApp, Dataset - it decides which
+resource will be deployed with specification listed below. `namespace` key gives information that identifies
+the deployment: state, context and platform. 
+
+Then, each file declares resource configuration under `spec` key, e.g., in organization deployment file:
```

Comment:
Then, each file declares resource configuration under `spec` key, specifically in the `payload` key, ....